doc/classes/AwaitTweener.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="AwaitTweener" inherits="Tweener" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Awaits a specified signal.
+	</brief_description>
+	<description>
+		[AwaitTweener] is used to await a specified signal, allowing asynchronous steps in [Tween] animation. See [method Tween.tween_await] for more usage information.
+		The [signal Tweener.finished] signal is emitted when either the awaited signal is received or when timeout is reached.
```

Comment:
I think? This is assuming it's going to be the behavior.

You may have purposely omitted this detail as it may be noted down elsewhere. However, the way this sentence is worded makes it sound like the condition at which `finished` is emitted has been overridden by this class.
```suggestion
		The [signal Tweener.finished] signal is emitted when either the awaited signal is received, when timeout is reached, or when the target object is freed.
```

Reply:
I added the last case just now. Previously it would not emit the signal.
I think we could add such note to all Tweeners.